core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/requirements.aarch64.txt

```diff
@@ -114,6 +114,7 @@ xorg-x11-font-utils # for grafana
 xorg-x11-server-utils # for grafana
 yum-plugin-versionlock
 yum-utils
+python3-3.6.8
```

Comment:
There is no newer package available? Python 3.6.x is deprecated as of 2021-12-23.

Reply:
This package is needed by pgbouncer. I've tried to use symlinks, but pgbouncer in the version for upgraded PostgreSQL is failing anyway during installation.

Reply:
Have you checked the latest version of PgBouncer? From `pgbouncer.spec` (master branch) it looks like PgBouncer requires `python3-psycopg2` in any version and there is no other dependencies directly related to Python.

Reply:
I have checked the last PgBouncer, it was failing with the python 3 installed from software collections even with created symlinks.